estate/models/estate_property_offer.py

```diff
@@ -14,17 +14,54 @@ class PropertyOffer(models.Model):
         ],
         copy=False,
     )
+    buyer_id = fields.Many2one("res.partner")
+    buyer_id = fields.Many2one("res.partner")
     partner_id = fields.Many2one("res.partner", required=True)
     property_id = fields.Many2one("property", required=True)
     validity = fields.Integer(compute="_compute_validity", inverse="_inverse_validity", default = 7)
-    date_deadline = fields.Datetime()
+    date_deadline = fields.Date(compute="_compute_date_deadline", inverse="_inverse_date_deadline")
+
+    @api.depends("validity")
+    def _compute_date_deadline(self):
+        for record in self:
+            if record.validity:
+                record.date_deadline = fields.Date.today() + timedelta(days=record.validity)
+
+    def _inverse_date_deadline(self):
+        for record in self:
+            if record.date_deadline:
```

Comment:
if there is no `date_deadline` this is not going to do anything, is this expected?